Sources/MapboxDirections/Incident.swift

```diff
@@ -25,94 +25,28 @@ public struct Incident: Codable, Equatable {
     public enum Kind: String {
-        case Accident = "accident"
-        case Congestion = "congestion"
-        case Construction = "construction"
-        case DisabledVehicle = "disabled_vehicle"
-        case LaneRestriction = "lane_restriction"
-        case MassTransit = "mass_transit"
-        case Miscellaneous = "miscellaneous"
-        case OtherNews = "other_news"
-        case PlannedEvent = "planned_event"
-        case RoadClosure = "road_closure"
-        case RoadHazard = "road_hazard"
-        case Weather = "weather"
-    }
-    
-    /// Defines a lane affected by the `Incident`
-    ///
-    /// Each `Incident` may have arbitrary of affected lanes
-    public enum BlockedLane: String, Codable {
-        /// Left lane
-        case left = "LEFT"
-        /// Left and center lanes
-        case leftCenter = "LEFT CENTER"
-        /// Left turn lane
-        case leftTurnLane = "LEFT TURN LANE"
-        /// Center lane
-        case center = "CENTER"
-        /// Right lane
-        case right = "RIGHT"
-        /// Right and center lanes
-        case rightCenter = "RIGHT CENTER"
-        /// Right turn lane
-        case rightTurnLane = "RIGHT TURN LANE"
-        /// High occupancy vehicle lane
-        case highOccupancyVehicle = "HOV"
-        /// Side lane
-        case side = "SIDE"
-        /// Shoulder lane
-        case shoulder = "SHOULDER"
-        /// Median lane
-        case median = "MEDIAN"
-        /// 1st Lane.
-        ///
-        /// Mind the driving side.
-        case lane1 = "1"
-        /// 2nd Lane.
-        ///
-        /// Mind the driving side.
-        case lane2 = "2"
-        /// 3rd Lane.
-        ///
-        /// Mind the driving side.
-        case lane3 = "3"
-        /// 4th Lane.
-        ///
-        /// Mind the driving side.
-        case lane4 = "4"
-        /// 5th Lane.
-        ///
-        /// Mind the driving side.
-        case lane5 = "5"
-        /// 6th Lane.
-        ///
-        /// Mind the driving side.
-        case lane6 = "6"
-        /// 7th Lane.
-        ///
-        /// Mind the driving side.
-        case lane7 = "7"
-        /// 8th Lane.
-        ///
-        /// Mind the driving side.
-        case lane8 = "8"
-        /// 9th Lane.
-        ///
-        /// Mind the driving side.
-        case lane9 = "9"
-        /// 10th Lane.
-        ///
-        /// Mind the driving side.
-        case lane10 = "10"
+        case accident = "accident"
+        case congestion = "congestion"
+        case construction = "construction"
+        case disabledVehicle = "disabled_vehicle"
+        case laneRestriction = "lane_restriction"
+        case massTransit = "mass_transit"
+        case miscellaneous = "miscellaneous"
+        case otherNews = "other_news"
+        case plannedEvent = "planned_event"
+        case roadClosure = "road_closure"
+        case roadHazard = "road_hazard"
+        case weather = "weather"
```

Comment:
These enumeration values should have at least pro-forma documentation comments so they appear in the docset.